components/overview/ContentAccordion.vue

```diff
@@ -27,10 +29,13 @@ import AccordionLayout from '~/components/overview/AccordionLayout.vue'
 export default class extends Vue {
   @Prop(Array) elements
 
-  expandedItem = 0
+  open = Array.from({ length: this.elements.length }, (_, i) => i === 0)
 
   actionChange (ev: any) : void {
```

Comment:
Non-blocking: the preferred abbreviation in HTML for the event is `evt`.

You can also annotate `ev` with type `{changedItem: number, state: boolean[]}`. You could even unpack both values here:
```suggestion
  actionChange ({ changedItem, state }: {changedItem: number, state: boolean[]}) : void {
```